rdrf/rdrf/forms/components.py

```diff
@@ -599,12 +602,17 @@ def __init__(self, user, registry_model, patient_model):
                 sorted([wg.name for wg in self.patient_model.my_index.working_groups.all()]))
         else:
             self.index_working_groups = None
+        fth = FormTitleHelper(self.registry_model, "Family linkage")
+        self.form_title_dict = fth.title_dict_for_user(self.user)
 
     def _get_template_data(self):
-        data = {"patient": self.patient_model,
-                "link_allowed": self.link_allowed,
-                "is_index": self.is_index,
-                "registry_code": self.registry_model.code,
-                "index_working_groups": self.index_working_groups}
+        data = {
+            "patient": self.patient_model,
+            "link_allowed": self.link_allowed,
+            "is_index": self.is_index,
+            "registry_code": self.registry_model.code,
+            "index_working_groups": self.index_working_groups,
+            "form_title_dict": self.form_title_dict
```

Comment:
Won't this line will fail if the FAMILY_LINKAGE feature isn't enabled?

Reply:
No.

Reply:
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/django/contrib/staticfiles/handlers.py", line 65, in __call__
    return self.application(environ, start_response)
  File "/usr/local/lib/python3.6/site-packages/django/core/handlers/wsgi.py", line 142, in __call__
    response = self.get_response(request)
  File "/usr/local/lib/python3.6/site-packages/django/core/handlers/base.py", line 78, in get_response
    response = self._middleware_chain(request)
  File "/usr/local/lib/python3.6/site-packages/django/core/handlers/exception.py", line 36, in inner
    response = response_for_exception(request, exc)
  File "/usr/local/lib/python3.6/site-packages/django/core/handlers/exception.py", line 90, in response_for_exception
    response = handle_uncaught_exception(request, get_resolver(get_urlconf()), sys.exc_info())
  File "/usr/local/lib/python3.6/site-packages/django/core/handlers/exception.py", line 125, in handle_uncaught_exception
    return debug.technical_500_response(request, *exc_info)
  File "/usr/local/lib/python3.6/site-packages/django_extensions/management/technical_response.py", line 37, in null_technical_500_response
    six.reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.6/site-packages/six.py", line 692, in reraise
    raise value.with_traceback(tb)
  File "/usr/local/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/local/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/django/views/generic/base.py", line 88, in dispatch
    return handler(request, *args, **kwargs)
  File "/app/rdrf/rdrf/views/patient_view.py", line 660, in get
    "family_linkage_panel": family_linkage_panel.html,
  File "/app/rdrf/rdrf/forms/components.py", line 62, in html
    return self._fill_template()
  File "/app/rdrf/rdrf/forms/components.py", line 69, in _fill_template
    data = self._get_template_data()
  File "/app/rdrf/rdrf/forms/components.py", line 615, in _get_template_data
    "form_title_dict": self.form_title_dict
AttributeError: 'FamilyLinkagePanel' object has no attribute 'form_title_dict'

Reply:
Updated.